File: hernandez_sebastian_a1/progExec1.java
public class ProgExec1 {

    public static void main(String[] args) {
        int dog = 5;
        for (int i = 0; i < 1; i++) {
            int cat = 6;
        }
        try {
            dog += cat; //dog is equal to 11
            System.out.println("Dog has value of " + dog);
        } catch (Exception e) {
            System.out.println("Dog has value of " + dog);
        }
    }
}

File: hernandez_sebastian_a1/progExec1.cpp
#include <iostream>

int main() {
    int dog = 5;
    for (int i = 0; i < 1; ++i) {
        int cat = 6;
    }
    try {
        dog += cat;
        std::cout << "Dog has a value of " << dog << std::endl; //will be 11 if cat has a scope that will allow it
    }  catch (int e){
        std::cout << "Dog has a value of " << dog << std::endl; //will be 5 if cat has no scope outside of the for loop
    }

    return 0;
}